master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/ha/StateMachine.java

```diff
@@ -207,6 +210,9 @@ private synchronized void loadSnapshot(SingleFileSnapshotInfo snapshot) throws I
     if (snapshot == null) {
       return;
     }
+    if (snapshot.getTermIndex().compareTo(getLastAppliedTermIndex()) <= 0) {
+      return;
```

Comment:
can we add a log here? if it's rarely happen, info is fine, otherwise debug is preferred

Reply:
Done. It's a rare situation, so I use info here.